coral-component-colorinput/src/scripts/ColorInputSwatch.js

```diff
@@ -39,6 +40,7 @@ const ColorInputSwatch = Decorator(class extends BaseColorInputAbstractSubview(B
     // Templates
     this._elements = {};
     colorButton.call(this._elements);
+    colorLabel.call(this._elements);
```

Comment:
Pass `commons` through to the `colorButton` template so that we can call generate a `uid` within the template.
```suggestion
    colorButton.call(this._elements, {commons});
    colorLabel.call(this._elements);
```